lib/turn.rb

```diff
@@ -0,0 +1,97 @@
+class Turn
+  attr_reader :player1, :player2, :spoils_of_war, :trashcan
+
+  def initialize(player1, player2)
+    @player1 = player1
+    @player2 = player2
+    @spoils_of_war = Deck.new([])
+    @trashcan = Deck.new([]) # cards to be deleted
+    @shadow_realm = Deck.new([])
+  end
+
+  # # a little cheeky alias perhaps
+  # def spoils_of_war
+  #   return @spoils_of_war.cards
+  # end
+
+  def type
+    @type = :basic
+    if @player1.deck.rank_of_card_at(0) == @player2.deck.rank_of_card_at(0)
+      @type = :war
+      if @player1.deck.cards.length >= 3 && @player2.deck.cards.length >= 3
+        if @player1.deck.rank_of_card_at(2) == @player2.deck.rank_of_card_at(2)
+          @type = :mutually_assured_destruction
+        end
+      end
+    end
+    return @type
```

Comment:
Think about how you might be able to accomplish the same outcome without nesting `if` statements.